test/extended/util/framework.go

```diff
@@ -2363,3 +2363,50 @@ func IsCapabilityEnabled(oc *CLI, cap configv1.ClusterVersionCapability) (bool,
 	}
 	return false, nil
 }
+
+// CheckPlatform check the cluster's platform
+func CheckPlatform(oc *CLI) string {
+	output, _ := oc.AsAdmin().WithoutNamespace().Run("get").Args("infrastructure", "cluster", "-o=jsonpath={.status.platformStatus.type}").Output()
+	return strings.ToLower(output)
+}
+
+// SkipTestIfSupportedPlatformNotMatched skip the test if supported platforms are not matched
+func SkipTestIfSupportedPlatformNotMatched(oc *CLI, supported ...string) {
```

Comment:
In https://github.com/openshift/origin/blob/master/test/extended/etcd/helpers/helpers.go#L526-L560 we have a set of skippers, I'd prefer we create a top-level ones, which will be `SkipIf(oc *CLI, platform... configv1.PlatformType)`, and slowly transition to those. While doing so make sure to update these as well.